src/stavenote.ts

```diff
@@ -569,7 +569,7 @@ export class StaveNote extends StemmableNote {
       if (lastLine === undefined) {
         lastLine = line;
       } else {
-        if (Math.abs(lastLine - line) === 0.5) {
+        if (Math.abs(lastLine - line) < 1) {
```

Comment:
Thanks for finding this.  wow.  Maybe good to add a comment like :`< 1 means == 0.5 (second) or 0.0 (unison)`

I totally see how this would fix the problem.  I just have no clue why it worked in the first place for stems up!

Reply:
With stems up the notehead is always displaced to the left